swap_meet/vendor.py

```diff
@@ -1,2 +1,67 @@
 class Vendor:
-    pass
+
+    def __init__(self, inventory=None):
+        if inventory is None:
+            self.inventory = []
+        else:
+            self.inventory = inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
+
+    def get_by_category(self, category):
+        items_in_same_category = []
+
+        for item in self.inventory:
+            if item.category == category:
+                items_in_same_category.append(item)
+
+        return items_in_same_category
```

Comment:
This looks great! We can also use a list comprehension here:

```suggestion
    def get_by_category(self, category):
        items_in_same_category = [item for item in self.inventory if item.category == category]
        return items_in_same_category
```

Here's more examples of Python list comprehensions if you want to continue playing around with them 😄 : https://www.w3schools.com/python/python_lists_comprehension.asp